### PR TITLE
Add very basic CI

### PR DESCRIPTION
**Pull request by almusil**
As a sanity check that everything still works add basic CI.
The goal is to build and start the cluster and within the
cluster check that the configuration works with very basic
tests.

Signed-off-by: Ales Musil <amusil@redhat.com>

### CODE REVIEW

**Comment by almusil** [comment]
@numansiddique @dceara Please take a look, I'll rebase it once #56 is merged, Successful run is here https://github.com/almusil/ovn-fake-multinode/actions/runs/2113182043

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
```
Comment (dceara) [inline .github/workflows/ci.yml:5]: Please use `main`.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+jobs:
+  basic:
+    env:
+      RUNC_CMD: podman
+      OVN_SRC_PATH: ovn_git
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download ovn-fake-multinode
+        uses: actions/checkout@v3
+      - name: Download OVN master
```
Comment (dceara) [inline .github/workflows/ci.yml:17]: Nit: s/master/main

**Comment by dceara** [comment]
Thanks for working on this @almusil, looks nice already!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (almusil) [reply .github/workflows/ci.yml]: Done, thanks

File: .github/workflows/ci.yml (hunk 2 above)
Comment (almusil) [reply .github/workflows/ci.yml]: Done, thanks

**Review by dceara — APPROVED** [review]
LGTM, thanks!

@numansiddique @flavio-fernandes does this look ok to you too?

**Comment by numansiddique** [comment]
LGTM to me too.

Thanks @almusil  for adding the CI job.  This is great.

**Pull request merged**